src/app/introduction/introduction.component.scss

```diff
@@ -0,0 +1,162 @@
+@import "../styles/components/_variables";
```

Comment:
this is very different from the `introduction.component.scss` in https://github.com/systers/PC-Prep-Kit/pull/25/files